### ISSUE TITLE
[FE] Featured story sort should match on system page and stories page

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-team#37095]
## User Story

As an editor I want the sort order of featured stories on the VAMC System page to match the display on the stories page so that Veterans have a consistent experience.

Currently the sorts do not match
![image](https://user-images.githubusercontent.com/5752113/154554998-3ae4cbf0-4d27-46e4-b319-f75e66d722b7.png)

System page: https://www.va.gov/puget-sound-health-care/
Stories list page: https://www.va.gov/puget-sound-health-care/stories/

Related slack conversation where it was raised by editor https://dsva.slack.com/archives/CT4GZBM8F/p1645123021852339

Second but related defect.   The featured section on the system page is not respecting the checkbox,  it is only using the 1st or second.
![image](https://user-images.githubusercontent.com/5752113/154589818-bd26e615-5c55-4913-b817-8dd80393b4e7.png)

- [ ] Attached to an Epic (what body of work is this a part of?)
- [ ] Correctly Labeled
- [ ] Fill in User Stories
- [ ] Fill in Documentation
- [ ] Fill in Tasks
- [ ] Fill in Acceptance Criteria

---

## Description

> this should be an understandable description of the ticket. This description should have enough context meaning that non-technical team members should get the idea.

## Tasks

- [ ] Decide which sort method makes the most sense to editors
- [ ] Make both templates use the same sort method.

## Acceptance Criteria

> when is the ticket done

- [ ] %Add any additional Acceptance Criteria%
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature

**Comment by swirtSJW** [comment]
Cc @mmiddaugh

**Comment by stefaniefgray** [comment]
Original ticket: VAHELP-3558

**Comment by swirtSJW** [comment]
It seems like the defect is largely on the template for featured stories on the system page 
It is only using the order and not using the featured flag and the sort is different from the story listing pages.

**Comment by stefaniefgray** [comment]
Reported by VA Minneapolis in VAHELP-3582 on 2/22/22

**Comment by stefaniefgray** [comment]
Per Jim H in VAHELP-3611:

**Update 1:**
"Obviously, there still are issues in the CMS with feature stories. I was going back through old stories, and it appears that the feature checkbox alone is not sufficient to control front page display. 

The story Equine-assisted therapy helps Veterans is one of two old stories that have republished to the front of the Salt Lake City home page. The feature box is unchecked, although the order still says second. I am going to change the order to none and republish to see if that deletes the story from the front of the site. If that works, I will go into all old features and change those designations and republish. 

That would not resolve the CMS issue, but might provide a temporary work around controllable at the local level."

**Update 2:**
"I manually republished all 39 Salt Lake City feature stories.. I didn’t count them all as I was flying through them quickly, but I would estimate that about half still had the order listed as first or second instead of none, even though the feature box was unchecked. 

This was how the home page changed while I was republishing – roughly 9:30 a.m.… so I’m guessing that it was in the midst of a publishing cycle.  

The second grab is the home page at about 10:30 a.m. which now shows the most recent feature – which is correct. In the list of stories, it now is the only story with the feature box checked and with a designated order (1st). 

I’m going to try that now with Montana, which also has its oldest stories on the website. 

It is cumbersome and time-consuming to go back and republish every story – especially for those facilities that produce a lot of content – but it seems to work. 

Once that is done, the best rule of thumb for now as a workaround, is when you post a new story, designate that as a feature and select the order; then go to the current stories and remove both the feature and order."

**Update 3:**
"I repeated the republishing process with all eight Montana feature stories. The ordering block was set to either first or second for four of the stories, instead of none. 

After unchecking the feature block designation and changing the order to none, as needed, the current stories correctly posted to the front. 

In addition to Salt Lake City, which I sent previously, and Montana – I also did this for five other facilities in VISN19 and it worked – although they produce far fewer features. 

As such, it appears that even if the feature box is unchecked, if the story order is not set to none, the story will go the home page. With the current CMS issue, for some reason that still is putting the oldest story with either the feature box checked or the order listed as 1st or 2nd.  

The easiest way until the CMS issue is fixed.. is uncheck those areas on any stories you don’t want featured, and check those for the one or two you want on the home page. When a new story is posted, simply go back and uncheck the recent ones you want removed. "
